specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/streamingjobs.json

```diff
@@ -639,10 +629,38 @@
           "readOnly": true,
           "type": "string",
           "description": "The current entity tag for the streaming job. This is an opaque string. You can use it to detect whether the resource has changed between requests. You can also use it in the If-Match or If-None-Match headers for write operations for optimistic concurrency."
+        },
+        "jobStorageAccount": {
+          "$ref": "#/definitions/JobStorageAccount"
+        },
+        "contentStoragePolicy": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Valid values are JobStorageAccount and SystemAccount. If set to JobStorageAccount, this requires the user to also specify jobStorageAccount property. .",
+          "enum": [
+            "SystemAccount",
+            "JobStorageAccount"
+          ],
+          "x-ms-enum": {
+            "name": "ContentStoragePolicy",
+            "modelAsString": true
+          }
+        },
+        "identity": {
+          "$ref": "./inputs.json#/definitions/Identity",
+          "description": "Describes how identity is verified."
+        },
+        "externals": {
+          "$ref": "#/definitions/External",
+          "description": "External Property."
+        },
+        "cluster": {
```

Comment:
this is an object

Reply:
Example of how it will look in the json:

"cluster": { 
   "id": "/subscriptions/409468b9-1047-4f2c-b98b-9245c3d1448d/resourceGroups/IoTResources/providers/Microsoft.StreamAnalytics/clusters/MyCluster" 
},

Reply:
Fixed